opsdroid/message.py

```diff
@@ -21,11 +22,31 @@ def __init__(self, text, user, room, connector):
         self.regex = None
         self.responded_to = False
 
+    async def _thinking_delay(self, seconds=0):
+        """Make opsdroid wait x-seconds before responding."""
+        seconds = self.connector.configuration.get('thinking-delay', 0)
+
+        if isinstance(seconds, list):
+            seconds = randrange(seconds[0], seconds[1])
+
+        await asyncio.sleep(seconds)
+
+    async def _typing_delay(self, text):
+        """Simulate typing, default is set to 6 characters per second."""
+        seconds = self.connector.configuration.get('typing-delay', 1)
```

Comment:
Why does this default to one character per second? I would expect that if it hasn't been configured there should be no delay. 

I would go for something like this and skip the sleep if it hasn't been configured.

```python
try:
    seconds = self.connector.configuration['typing-delay']
    char_count = len(text)
    await asyncio.sleep(char_count//seconds)
except KeyError:
    pass
```